### ISSUE TITLE
[BUG] - Too many error messages if SkyQ is not powered on HA start

### ISSUE THREAD

**Issue by snakuzzo** [RogerSelwyn/Home_Assistant_SkyQ_MediaPlayer#22]
**Describe the bug**  
A lot of error on messages if skyq is not reachable when HA starts
When I start HA with latest 2.3.5 version and skyq is not connected, on HA log there are a lot of errors repeated about 20 times before it stops to log.
HA start finishes only when your component stops logging.

**To Reproduce**  
1. What were you doing/watching at the time ? nothing
2. What channel were you on? nothing
3. What time did the problem occur? every time HA starts using 2.3.5 version
4. Where are you (bear in mind component supports GBR/ITA at the moment)? ITA

**Expected behavior**  
Just few checks on skyq availability to start HA quickly and reduced log errors

**Screenshots**  

**Component versions**  
1. What version of SkyQ component are you using? 2.3.5
2. What was the last version where the problem was not shown? 2.3.4

**Your configuration**  
How have you setup your devices, by UI or YAML? YAML

**Your configuration from the options dialogue**  
or  
**Your configuration.yaml**  
```yaml
media_player:
- platform:  skyq
  name: Sky
  host: 192.168.1.15
  #output_programme_image: False
  live_tv: False
  country: "ITA"
  #generate_switches_for_channels: true
  sources:
    SkyTG24HD_100: '1,0,0'
    Rai1_101: '1,0,1'
    Rai2_102: '1,0,2'
    Rai3_103: '1,0,3'
    ...
````
**Output of HA logs**  
```
2020-08-15 12:04:51 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for skyq which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
2020-08-15 12:05:10 WARNING (SyncWorker_28) [pyskyqremote.skyq_remote] W0010 - Control URL not accessible: 192.168.1.15 : http://192.168.1.15:49153/description0.xml
2020-08-15 12:05:11 ERROR (SyncWorker_28) [pyskyqremote.skyq_remote] X0060 - Connection error: 192.168.1.15 : HTTPConnectionPool(host='192.168.1.15', port=49153): Max retries exceeded with url: /description1.xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x4f5d66f0>: Failed to establish a new connection: [Errno 113] No route to host'))
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/connection.py", line 160, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
OSError: [Errno 113] No route to host

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/connectionpool.py", line 677, in urlopen
    chunked=chunked,
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/connectionpool.py", line 392, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.7/http/client.py", line 1244, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.7/http/client.py", line 1290, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.7/http/client.py", line 1239, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.7/http/client.py", line 1026, in _send_output
    self.send(msg)
  File "/usr/lib/python3.7/http/client.py", line 966, in send
    self.connect()
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/connection.py", line 187, in connect
    conn = self._new_conn()
  File "/srv/homeassistant/lib/python3.7/site-packages/urllib3/connection.py", line 172, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x4f5d66f0>: Failed to establish a new connection: [Errno 113] No route to host
```

**Additional context**

**Comment by RogerSelwyn** [comment]
Closed with 2.3.6

**Comment by snakuzzo** [comment]
Thank you ;)
